modules/processing/snowflake/sql/_VORONOIHELPER.sql

```diff
@@ -0,0 +1,47 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+
+CREATE OR REPLACE FUNCTION @@SF_PREFIX@@processing._VORONOIHELPER
+(geojson ARRAY, bbox ARRAY, typeOfVoronoi STRING)
+RETURNS ARRAY
+LANGUAGE JAVASCRIPT
+AS $$
+    @@SF_LIBRARY_CONTENT@@
+
+    if (!GEOJSON) {
+        return null;
+    }
+    
+    if (BBOX != null && BBOX.length != 4) {
+        throw new Error('Incorrect bounding box passed to UDF. It should contain the BBOX extends, i.e., [xmin, ymin, xmax, ymax]');
+    }
+
+    const options = {};
+
+    // If the BBOX parameter is not included, turf.js will use a default [-180,-85,180,-85] BBOX 
+    if (BBOX != null) {
```

Comment:
So here we would remove this check and add it to the top. And we will return null if they pass a null.

Reply:
Notice that this also might affect your tests, for references there are some tests to check default params in other functions as ST_MAKEELLIPSE.

Reply:
Ok!